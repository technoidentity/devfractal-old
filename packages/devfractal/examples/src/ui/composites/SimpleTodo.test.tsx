import React from 'react'
import renderer, { ReactTestRenderer } from 'react-test-renderer'
import { SimpleTodo } from './SimpleTodo'

test('Simple Todo', () => {
  const component: ReactTestRenderer = renderer.create(<SimpleTodo />)
  expect(component.toJSON()).toMatchSnapshot()
})
