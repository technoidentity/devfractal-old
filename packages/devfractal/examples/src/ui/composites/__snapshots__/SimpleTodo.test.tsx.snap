// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Simple Todo 1`] = `
Array [
  <section
    className="section"
  >
    <form
      action="#"
      onReset={[Function]}
      onSubmit={[Function]}
    >
      <div
        className="field"
      >
        <label
          className="label"
          htmlFor="title"
        >
          Title:
        </label>
        <div
          className="control"
        >
          <input
            className="input"
            id="title"
            name="title"
            onBlur={[Function]}
            onChange={[Function]}
            type="text"
            value=""
          />
        </div>
      </div>
      <div
        className="field"
      >
        <label
          className="label"
          htmlFor="description"
        >
          Description:
        </label>
        <div
          className="control"
        >
          <textarea
            className="textarea"
            id="description"
            name="description"
            onBlur={[Function]}
            onChange={[Function]}
            value=""
          />
        </div>
      </div>
      <div
        className="field"
      >
        <label
          className="label"
          htmlFor="done"
        >
           Done
        </label>
        <div
          className="control"
        >
          <label
            className="checkbox"
          >
            <input
              checked={false}
              className="checkbox"
              id="done"
              name="done"
              onBlur={[Function]}
              onChange={[Function]}
              type="checkbox"
              value={false}
            />
          </label>
        </div>
      </div>
      <div
        className="buttons"
      >
        <button
          className="button is-info"
          disabled={false}
          type="submit"
        >
          Submit
        </button>
        <button
          className="button is-danger"
          disabled={true}
          onClick={[Function]}
          type="reset"
        >
          Reset
        </button>
      </div>
      <pre>
        {
  "title": "",
  "description": "",
  "done": false
}
      </pre>
    </form>
  </section>,
  <section
    className="section"
  >
    <div
      className="columns"
    >
      <div
        className="column"
      >
        <h1
          className="title is-6"
        >
          Id
        </h1>
      </div>
      <div
        className="column"
      >
        1
      </div>
    </div>
    <div
      className="columns"
    >
      <div
        className="column"
      >
        <h1
          className="title is-6"
        >
          Title
        </h1>
      </div>
      <div
        className="column"
      >
        bring milk
      </div>
    </div>
    <div
      className="columns"
    >
      <div
        className="column"
      >
        <h1
          className="title is-6"
        >
          Done
        </h1>
      </div>
      <div
        className="column"
      >
        <div
          className="control"
        >
          <label
            className="checkbox"
          >
            <input
              checked={true}
              className="checkbox"
              readOnly={true}
              type="checkbox"
            />
          </label>
        </div>
      </div>
    </div>
  </section>,
  <table
    className="table is-fullwidth"
  >
    <thead
      className=""
    >
      <tr
        className="tr"
      >
        <th
          className=""
        >
          Id
        </th>
        <th
          className=""
        >
          Title
        </th>
        <th
          className=""
        >
          Done
        </th>
      </tr>
    </thead>
    <tbody
      className=""
    >
      <tr
        className="tr"
        onClick={[Function]}
      >
        <td
          className=""
        >
          1
        </td>
        <td
          className=""
        >
          bring milk
        </td>
        <td
          className=""
        >
          <div
            className="control"
          >
            <label
              className="checkbox"
            >
              <input
                checked={true}
                className="checkbox"
                readOnly={true}
                type="checkbox"
              />
            </label>
          </div>
        </td>
      </tr>
      <tr
        className="tr"
        onClick={[Function]}
      >
        <td
          className=""
        >
          2
        </td>
        <td
          className=""
        >
          go for a walk
        </td>
        <td
          className=""
        >
          <div
            className="control"
          >
            <label
              className="checkbox"
            >
              <input
                checked={false}
                className="checkbox"
                readOnly={true}
                type="checkbox"
              />
            </label>
          </div>
        </td>
      </tr>
      <tr
        className="tr"
        onClick={[Function]}
      >
        <td
          className=""
        >
          3
        </td>
        <td
          className=""
        >
          read news
        </td>
        <td
          className=""
        >
          <div
            className="control"
          >
            <label
              className="checkbox"
            >
              <input
                checked={false}
                className="checkbox"
                readOnly={true}
                type="checkbox"
              />
            </label>
          </div>
        </td>
      </tr>
    </tbody>
  </table>,
]
`;
